addons/core/defines.hpp

```diff
@@ -13,6 +13,7 @@
 #define RADIO_OWNER 7
 #define SPEAKER_OFFSET 8
 #define POWER_OFFSET 9
+#define HALFDUPLEX_OVERRIDE_OFFSET 10
```

Comment:
You are doing everything correct here. But last time we added an element to that Array some Mods had problems. Just need to make it public that the first 30 are reserved for us.